src/rpc/rpc-handlers.ts

```diff
@@ -275,7 +277,9 @@ const handleRPC: HandlerFunc = async data => {
             case 'solana':
                 result = await solana.send({method, params, id, network});
                 break;
+            case 'cosmos':
             case 'keplr':
```

Comment:
I had to add `cosmos` as an option here because the network in the request from Point SDK is actually `cosmos_testnet` (which has `type: cosmos`).

I would suggest we remove `keplr` here, but didn't want to do it just yet, as it will probably break when running in non-development mode, since `cosmos_testnet` only exists in the development config. I created ticket [PN-286](https://point-labs.atlassian.net/browse/PN-286) to deal with this.